pkg/chart/models/chart.go

```diff
@@ -71,9 +71,11 @@ type ChartVersion struct {
 	Created    time.Time `json:"created"`
 	Digest     string    `json:"digest"`
 	URLs       []string  `json:"urls"`
-	Readme     string    `json:"readme" bson:"-"`
-	Values     string    `json:"values" bson:"-"`
-	Schema     string    `json:"schema" bson:"-"`
+	// TODO(mnelson): Why are the following three fields repeated here, when they are
+	// already on the ChartFiles struct?
```

Comment:
I _think_ these values are not really being used, whenever the Readme or the values are needed, the frontend does a specific request and it uses the ChartFiles instead. This was present in the code of the chartsvc and I couldn't verify that.

Reply:
I tried deleting them to see what would happen: it turns out they are storing the URL paths to these items for when the handler returns a chart version (see handler.go's `chartVersionAttributes`). I've updated the comment so that's obvious to the next person.

Reply:
ah, true. In any case, I don't think those URLs are necessary (the path is always the same)